yaml/_dumper_state.ts

```diff
@@ -870,9 +874,7 @@ export class DumperState {
       } else {
         if (this.skipInvalid) return null;
         throw new TypeError(
-          `unacceptable kind of an object to dump ${
-            getObjectTypeString(value)
-          }`,
+          `Cannot dump object of type: ${getObjectTypeString(value)}`,
```

Comment:
The verb 'dump' is only used internally. The public API is named `stringify`. So let's use 'stringify' here.

```suggestion
          `Cannot stringify object of type: ${getObjectTypeString(value)}`,
```